src/blog/2023/06/node-red-as-a-no-code-ethernet_ip-to-s7-protocol-converter.md

```diff
@@ -189,7 +189,7 @@ Depending on how noisy the REAL data is, which is common with unfiltered 4-20mA
 
 ![Filter node Configuration](./images/ethip-to-S7/e-to-p-23.png)
 
-In the example above, we arbitrarily applied a 3% [deadband](/node-red/core-nodes/rbe/#deadband-mode)
+In the example above, we arbitrarily applied a 3% [deadband](/node-red/core-nodes/filter/#deadband-mode)
```

Comment:
I suppose this would work on this PR: https://github.com/FlowFuse/website/pull/1956, but it's currently causing an error in the tests here. Should we wait until that PR is merged? Or would you prefer to include it as part of that PR instead?